README.md

```diff
@@ -260,6 +260,13 @@ $ mkuimage -files "$HOME/hello.ko:etc/hello.ko" -files "$HOME/hello2.ko:etc/hell
 $ qemu-system-x86_64 -kernel /boot/vmlinuz-$(uname -r) -initrd /tmp/initramfs.linux_amd64.cpio
 ```
 
+Use `-skip-ldd` to not automatically include ldd dependencies for binary files. This can be useful for
+- Reproducible Builds: Ensures builds don't depend on the host system's libraries
+- Cross-compilation: When host libraries are incompatible with target architecture
+- Controlled Dependencies: When you want to manually specify exact library versions
+- Static Binaries: When dealing with statically linked binaries that don't need shared libraries
```

Comment:
I think Static binaries are kind of not important in this case: if they are static, it won't find any shared libraries. I feel like line 267 is therefore a bit confusing.